style.css

```diff
@@ -0,0 +1,87 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  font-family: 'Poppins', sans-serif;
+}
+
+body {
+  position: relative;
+  background-color: #fff;
+}
+
+header {
+  color: #6070ff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1em 7%;
+}
+
+.logo > a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.headline-main {
+  max-height: 800px;
+  min-height: 550px;
+  width: auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 0 7%;
+  background-image: url(./assets/headline-background-mobile.jpg);
```

Comment:
-[OPTIONAL] Your background image is repeating horizontally, kindly consider adding `background-repeat: no-repeat;` here. and check out this link for more info:
[Background-Image-No-Repeat](https://www.w3schools.com/css/tryit.asp?filename=trycss_background-image_norepeat)